index.html

```diff
@@ -33,6 +33,258 @@
 </head>
 
 <body>
+  <section id="abstract">
+    <h2>Abstract</h2>
+    <p>
+      This document defines a new API {{MediaDevices/getAllScreensMedia}}
+      for capturing multiple
+      <a data-cite="screen-capture#dfn-display-surface">monitors</a>
+      following one user gesture.
+      It is an extension to the Screen Capture API [[screen-capture]].
+    </p>
+  </section>
+  <section id="sotd">
+    <p>
+      This document is a draft. It is subject to major changes and, while early
+      experimentations are encouraged, it is therefore not intended for implementation.
+    </p>
+  </section>
+  <section class="informative" id="introduction">
+    <h2>
+      Introduction
+    </h2>
+    <p>
+      The Screen Capture API [[screen-capture]] enables the capturing of a single
+      <a data-cite="screen-capture#dfn-display-surface">display surface</a> in the
+      form of a video track.
+    </p>
+    <p>
+
+      Users currently cannot capture all
+      <a data-cite="screen-capture#dfn-monitor">monitors</a> attached to a device
+      at once without having to execute a two step sequence multiple times,
+      i.e. an interaction with the page through a [=transient activation=] that calls
+      <a data-cite="screen-capture#dom-mediadevices-getdisplaymedia">getDisplayMedia</a>
+      and a subsequent interaction with the media picker.
+      This is a cumbersome process and does not guarantee that all
+      <a data-cite="screen-capture#dfn-monitor">monitors</a> are captured
+      (which may be required by regulations).
+    </p>
+    <p>
+      This document describes {{MediaDevices/getAllScreensMedia}}, an extension to
+      the Screen Capture API [[screen-capture]].
+      {{MediaDevices/getAllScreensMedia}} enables
+      the capturing of several of the user's <a data-cite="screen-capture#dfn-monitor">monitors</a>,
+      or parts thereof, without a [=transient activation=].
+      It returns a list of media streams
+      (each containing a track corresponding to the captured
+      <a data-cite="screen-capture#dfn-monitor">monitor</a>).
+      As no [=transient activation=] is required, the <a href="#dfn-user-agent">user agent</a>
+      MUST ensure that appropriate protection, e.g. by allowlisting permitted
```

Comment:
This is not a normative section, so I don't think it's customary to capitalize MUST here.